src/encoded/tests/test_audit_experiment.py

```diff
@@ -3944,3 +3944,14 @@ def test_audit_experiment_analysis_files(
         error['category'] == 'inconsistent analysis files'
         for error in collect_audit_errors(res)
     )
+
+
+def test_audit_experiment_average_fragment_size(testapp, base_experiment, base_replicate, base_library):
+    # average_fragment_size may stand in for size_range, behavior should match
+    testapp.patch_json(base_experiment['@id'], {'assay_term_name': 'RNA-seq'})
+    testapp.patch_json(base_library['@id'], {'average_fragment_size': 220})
+    testapp.patch_json(base_replicate['@id'], {'library': base_library['@id']})
+    res = testapp.get(base_experiment['@id'] + '@@index-data')
+    assert any(error['category'] == 'missing spikeins'
+               for error in collect_audit_errors(res))
+    assert 'missing RNA fragment size' not in collect_audit_errors(res)
```

Comment:
calling twice to collect_audit_errors, perhaps it is more efficient to define a variable and use it twice?

Reply:
I updated this to check a new variable res_errors instead of collecting the errors twice